contrib/exporters/secadvisor/mod/encode.go

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2019 IBM, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy ofthe License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specificlanguage governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package mod
+
+import (
+	"github.com/spf13/viper"
+
+	"github.com/skydive-project/skydive/contrib/exporters/core"
+)
+
+type encodeSecadvisorJSON struct {
```

Comment:
you can just call it `encode` as:
(1) it already is part of secadvisor code base
(2) today it is JSON but tomorrow it may change something else (CSV, Parquet)

Reply:
done.